torch/_refs/__init__.py

```diff
@@ -3614,6 +3614,23 @@ def repeat(a: Tensor, *repeat_shape) -> Tensor:
     return permuted_result.reshape(target_shape)
 
 
+def _nd_to_1d(a: Tensor) -> Tensor:
+    torch._check(a.numel() > 0)
+    torch._check(a.ndim > 0)
+    torch._check(a.is_contiguous())
+    return torch.as_strided(a, [a.numel()], [1])
```

Comment:
Need to handle non-zero storage offset here (and below) correctly

Reply:
Hmm. I read this [documentation](https://pytorch.org/docs/stable/generated/torch.as_strided.html#torch-as-strided) which suggests that if I don't explicitly specify the optional storage_offset, then the storage_offset for the output will match that of the input. Is that not what we want?